template/cordova/lib/deployment.js

```diff
@@ -0,0 +1,272 @@
+/*
+       Licensed to the Apache Software Foundation (ASF) under one
+       or more contributor license agreements.  See the NOTICE file
+       distributed with this work for additional information
+       regarding copyright ownership.  The ASF licenses this file
+       to you under the Apache License, Version 2.0 (the
+       "License"); you may not use this file except in compliance
+       with the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+       Unless required by applicable law or agreed to in writing,
+       software distributed under the License is distributed on an
+       "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+       KIND, either express or implied.  See the License for the
+       specific language governing permissions and limitations
+       under the License.
+*/
+
+var Q     = require('q'),
+    fs    = require('fs'),
+    /* jshint ignore:start */ // 'path' only used in ignored blocks 
+    path  = require('path'),
+    /* jshint ignore:end */
+    proc  = require('child_process');
+
+// neither 'exec' nor 'spawn' was sufficient because we need to pass arguments via spawn
+// but also need to be able to capture stdout / stderr
+function run(cmd, args, opt_cwd) {
```

Comment:
True.  We could create a separate jira (I don't want to dot out of cordova-windows to access superspawn here) to do this refactor.